benchmark/sysbench.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Performace test result for TiDB using Sysbench
+category: benchmark 
+---
+
+# Performace test result for TiDB using Sysbench
+
+## Test purpose
+
+The purpose of this test is to test the performance and horizontal scalability of TiDB in OLTP scenarios.
+
+> **Note**: The results of the testing might vary based on different environmental dependencies.
+
+## Test version, date and place
+
+TiDB version: v1.0.0
+
+Date: October 20, 2017
+
+Place: Beijing
+
+## Test environment
+
+- IDC machines:
+
+  | Category  |  Detail       |  
+  | :--------| :---------| 
+  | OS       | Linux (CentOS 7.3.1611)       |   
+  | CPU | 40 vCPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz |
+  | RAM | 128GB | 
+  | DISK | 1.5T SSD * 2  + Optane SSD * 1 |  
+
+- Sysbench version: 1.0.6
+
+- Test script: https://github.com/pingcap/tidb-bench/tree/cwen/not_prepared_statement/sysbench.   
+
+## Test scenarios
+
+### Scenario one: simple performance test using Sysbench
+
+The structure of the table used for the test:
+
+``` sql
+CREATE TABLE `sbtest` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `k` int(10) unsigned NOT NULL DEFAULT '0',
+  `c` char(120) NOT NULL DEFAULT '',
+  `pad` char(60) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`),
+  KEY `k_1` (`k`)
+) ENGINE=InnoDB  
+``` 
+
+The deployment and configuration details:
+
+```
+// TiDB deployment
+172.16.20.4    4*tikv    1*tidb    1*sysbench
+172.16.20.6    4*tikv    1*tidb    1*sysbench
+172.16.20.7    4*tikv    1*tidb    1*sysbench
+172.16.10.8    1*tidb    1*pd      1*sysbench
+
+// Each physical node has three disks.
+data3: 2 tikv (Optane SSD) 
+data2: 1 tikv
+data1: 1 tikv
+
+// TiKV configuration
+sync-log = false
+grpc-concurrency = 8
+grpc-raft-conn-num = 24 
+[defaultcf]
+block-cache-size = "12GB"
+[writecf]
+block-cache-size = "5GB"
+[raftdb.defaultcf]
+block-cache-size = "2GB"
+
+// MySQL deployment
+// Use the semi-synchronous replication and asynchronous replication to deploy two replicas respectively.
+172.16.20.4    master    
+172.16.20.6    slave        
+172.16.20.7    slave
+172.16.10.8    1*sysbench 
+Mysql version: 5.6.37
+
+// MySQL configuration
+thread_cache_size = 64
+innodb_buffer_pool_size = 64G
+innodb_file_per_table = 1
+innodb_flush_log_at_trx_commit = 0  
+datadir = /data3/mysql  
+max_connections = 2000 
+```
+
+- Simple OLTP test
```

Comment:
How about `OLTP RW Test`?  I don't think `Simple` is suitable.

Reply:
you can refer to this [Percona Database Performance Blog](https://www.percona.com/blog/2017/01/06/millions-queries-per-second-postgresql-and-mysql-peaceful-battle-at-modern-demanding-workloads/)